scrapy/exporters.py

```diff
@@ -144,8 +144,10 @@ def _export_xml_field(self, name, serialized_value):
         elif is_listlike(serialized_value):
             for value in serialized_value:
                 self._export_xml_field('value', value)
-        else:
+        elif isinstance(serialized_value, (six.text_type, bytes)):
```

Comment:
Yeah, thanks for the tip. I updated the PR, but something is wrong with my history. I will check it.